### PR TITLE
Fixes attributes counting as 1

### PR DESCRIPTION
**Pull request by PotatoTomahto**
## About The Pull Request
- Renames ```level_bonus``` to ```stat_bonus```
- Attribute getters now return a minimum of 0 instead of 1

## Why It's Good For The Game
Renames the variable because it's just worded so confusingly. These changes fixes people with 0 stats having 101 max HP and sanity. 

## Changelog
:cl:
fix: attribute getters now return at minimum 0, 
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/abnormality/teth/dingle_dangle.dm [hunk 1]
```diff
@@ -35,31 +36,31 @@
...
 		user.apply_status_effect(STATUS_EFFECT_DANGLE)
 		datum_reference.qliphoth_change(-1)
 		return ..()
 
-	if(get_attribute_level(user, FORTITUDE_ATTRIBUTE) <= 80)
+	if(get_attribute_level(user, FORTITUDE_ATTRIBUTE) >= 80) // fort 4 or higher
 		return ..()
 
 	//I mean it does this in wonderlabs
-	user.dust()
```
Comment (Kitsunemitsu) [inline code/modules/mob/living/simple_animal/abnormality/teth/dingle_dangle.dm:48]: Why are you removing this?

File: code/modules/paperwork/records/info/teth.dm [hunk 2]
```diff
@@ -51,10 +51,11 @@
...
 	abno_code = "T-04-111"
 	abno_info = list(
-		"When employees with Prudence Level 3 or higher completed their work, they were immediately consumed by Dingle-Dangle, unless their Fortitude was Level 4 or above.",
-		"When employees with Prudence Level 2 or lower completed work with Dingle-Dangle, the Qliphoth counter lowered.",
-		"When employees with Prudence Level 2 or lower completed work with Dingle-Dangle, they reported heightend combat abilities.",
-		"When the work result was Bad, the employee was consumed by Dingle-Dangle with a normal probability.")
+		"When the work result was Bad, the employee was immediately consumed by Dingle-Dangle with a normal probability.",
+		"When employees with Prudence Level 3 or higher completed their work, they were consumed by Dingle-Dangle unless their Fortitude was Level 4 or higher.",
+		"When employees with Prudence Level 2 or lower completed their work with Dingle-Dangle, the Qliphoth Counter decreased.",
+		"These employees reported sensory hallucinations as well heightened combat capabilities.",
```
Comment (Kitsunemitsu) [inline code/modules/paperwork/records/info/teth.dm:57]: Do NOT state hallucinations on this, Ever. 
Hallucinations are extremely minor and rely on the players not knowing it's happening to have any effect at all.

File: code/modules/mob/living/simple_animal/abnormality/teth/dingle_dangle.dm (hunk 1 above)
Comment (Kitsunemitsu) [inline code/modules/mob/living/simple_animal/abnormality/teth/dingle_dangle.dm:48]: This is fundamentally changed and was not put in the PR.

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
Atomize the Dangle Changes, I will not approve balance changes in this PR.

**Comment by Kitsunemitsu** [comment]
Atomization Requested, these two things have nothing in common.

**Review by EgorDinamit — APPROVED** [review]
Make sure no divisions by 0 occur. That was the main reason to max it at 1.

**Pull request merged**